velox/docs/spark_functions.rst

```diff
@@ -13,6 +13,7 @@ Spark Functions
     functions/spark/datetime
     functions/spark/array
     functions/spark/map
+    functions/spark/misc_function
```

Comment:
misc_function -> misc (function is implied)

Reply:
Upated, thanks.